api/allocations.go

```diff
@@ -86,6 +86,7 @@ type Allocation struct {
 	EvalID                string
 	Name                  string
 	NodeID                string
+	NodeName              string
 	JobID                 string
```

Comment:
Needs a rebase to master to remove these unrelated changes that have already been merged